limacharlie/artifact.go

```diff
@@ -71,9 +71,14 @@ func (org Organization) ArtifactRuleDelete(ruleName ArtifactRuleName) error {
 	return nil
 }
 
-func (org Organization) ExportArtifact(artifactID string, deadline time.Time) (io.ReadCloser, error) {
+func (org Organization) ExportArtifact(artifactID string, deadline time.Time, optParams *interface{}) (io.ReadCloser, error) {
```

Comment:
```suggestion
func (org Organization) ExportArtifact(artifactID string, deadline time.Time, optParams Dict{}) (io.ReadCloser, error) {
```
A simple Dict of key-val will do. Also note that `interface{}` (an interface) is always just a referemce/pointer to a value, so the `*` is superfluous.